app/src/main/res/drawable-night/ic_all_tabs.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Nice, thanks for adding this.

Reply:
Please also include the license

Reply:
Good catch.  I don't think my original icon have it either.  Since we need this change in, @MozillaNoah can you please create another PR to add license on both `ic_all_tabs`?  Thanks